js/models/media_overlay.js

```diff
@@ -121,14 +165,20 @@ var MediaOverlay = function(package) {
         return this.smil_models[smilIndex];
     }
     
+    /**
+     * Calculates a percent of the total audio duration corresponding to a timecode
+     * Note: never called; obsolete?
```

Comment:
This is used here: https://github.com/readium/readium-js-viewer/blob/develop/src/js/EpubReaderMediaOverlays.js#L505